phabfive/maniphest.py

```diff
@@ -81,10 +96,149 @@ def create_from_config(self, config_file, dry_run=False):
                     has_errors = True
 
             if has_errors:
-                log.error(f"Could not find one or more specified projects in phabricator instance. All available project names")
+                log.error(f"Could not find one or more specified users, projects in phabricator instance. All available project names")
                 log.error(list(project_name_to_id_mapping.keys()))
                 return (False, None)
 
+        ### Validate owner and all subscribers (=users) is correct data types and exists in Phabricator instance
+        #create a new construct to consolidate all users for owner and subscriber
+        users_query = {}
+        users_query["data"] = []
+
+        r = self.phab.user.search()
+        length = len(r.data)
+        afterCursor = r['cursor']['after']
+        users_query["data"].extend(r.data)
+
+        while (length >= 100 and afterCursor is not None):
+            r = self.phab.user.search(
+                after=afterCursor)
+            length = len(r.data)
+            afterCursor = r['cursor']['after']
```

Comment:
Don't mix java and python standards, we use snake-case so this would be `after_cursor` please update all changes variables that this rule would apply to

Reply:
changed to follow what you(?) did for project.